codes/quantum/qubits/permutation_invariant.yml

```diff
@@ -0,0 +1,34 @@
+#######################################################
```

Comment:
This is a good umbrella entry for perm.-inv codes. What I had in mind was actually a specific entry for Ouyang's codes arXiv:1302.3247 (name "GNU permutation-invariant code"). In there, you can express them in terms of Dicke states (we have a way to do display equations now), mention the realization, and also mention the connections found in an appx of https://arxiv.org/abs/1708.05010. I can help fill this in, but was hoping you could give it a shot with a reasonably thorough initial draft.

Reply:
happy to do so